packages/blast-cmd/node/node.js

```diff
@@ -1,17 +1,35 @@
-const { executeCompose } = require('../../blast-utilities/run-docker-commands')
+const { executeCompose, executeAttach } = require('../../blast-utilities/run-docker-commands')
 const {
   getNodeStatus,
   checkNodeOnline,
   checkNodeOffline
 } = require('../../blast-utilities/get-node-status')
+const { getAdditionalAccounts } = require('../../blast-utilities/config-utils')
+const { handleAdditionalAccountCreation } = require('../../blast-utilities/keypair')
+const { delay } = require('../../blast-utilities/delay')
 
 const startNodeCmd = async function(argv) {
   await checkNodeOffline()
-  if (argv.daemon) {
-    executeCompose('up --build -d')
-    return
+
+  executeCompose('up --build -d')
+
+  while (true) {
```

Comment:
just a thought: now as we are detached from docker, if an error occurs during building the node, we would be stuck in this while loop.

Reply:
check the last commit please

Reply:
that was my first approach, but you can't `while (await getNodeStatus().isConnected)`.